packages/matter.js/src/cluster/definitions/WindowCoveringCluster.ts

```diff
@@ -0,0 +1,934 @@
+     * for commands (e.g: Open, Close, GoTos) to match the physical installation conditionsThis bit can be adjusted by
+     * setting the appropriate reversal bit value in the Mode attribute
+     */
+    liftMovementReversed: BitFlag(2),
+
+    /**
+     * Control - Lift: This status bit identifies if the window covering supports the Position Aware Lift Control
+     */
+    liftPositionAware: BitFlag(3),
+
+    /**
+     * Control - Tilt: This status bit identifies if the window covering supports the Position Aware Tilt Control
+     */
+    tiltPositionAware: BitFlag(4),
+
+    /**
+     * Encoder - Lift: This status bit identifies if a Position Aware Controlled Window Covering is employing an
+     * encoder for positioning the height of the window covering.
+     */
+    liftEncoderControlled: BitFlag(5),
+
+    /**
+     * Encoder - Tilt: This status bit identifies if a Position Aware Controlled Window Covering is employing an
+     * encoder for tilting the window covering.
+     */
+    tiltEncoderControlled: BitFlag(6)
+};
+
+/**
+ * The value of the WindowCovering operationalStatus attribute
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_1} § 5.3.5.15
+ */
+export const OperationalStatus = {
+    /**
+     * Movement status of the cover
+     */
+    global: BitFieldEnum<MovementStatus>(0, 2),
+
+    /**
+     * Movement status of the cover's lift function
+     */
+    lift: BitFieldEnum<MovementStatus>(2, 2),
+
+    /**
+     * Movement status of the cover's tilt function
+     */
+    tilt: BitFieldEnum<MovementStatus>(4, 2)
+};
+
+/**
+ * Values for OperationalStatus attribute fields.
+ */
+export const enum MovementStatus {
+    /**
+     * Covering is not moving
+     */
+    Stopped = 0,
+
+    /**
+     * Covering is moving from closed to open
+     */
+    Opening = 1,
+
+    /**
+     * Covering is moving from open to closed
+     */
+    Closing = 2
+}
+
+/**
+ * The value of the WindowCovering endProductType attribute
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_1} § 5.3.5.16
+ */
+export const enum EndProductType {
+    RollerShade = 0,
+    RomanShade = 1,
+    BalloonShade = 2,
+    WovenWood = 3,
+    PleatedShade = 4,
+    CellularShade = 5,
+    LayeredShade = 6,
+    LayeredShade2D = 7,
+    SheerShade = 8,
+    TiltOnlyInteriorBlind = 9,
+    InteriorBlind = 10,
+    VerticalBlindStripCurtain = 11,
+    InteriorVenetianBlind = 12,
+    ExteriorVenetianBlind = 13,
+    LateralLeftCurtain = 14,
+    LateralRightCurtain = 15,
+    CentralCurtain = 16,
+    RollerShutter = 17,
+    ExteriorVerticalScreen = 18,
+    AwningTerrace = 19,
+    AwningVerticalScreen = 20,
+    TiltOnlyPergola = 21,
+    SwingingShutter = 22,
+    SlidingShutter = 23,
+    Unknown = 255
+}
+
+/**
+ * The value of the WindowCovering mode attribute
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_1} § 5.3.5.21
+ */
+export const Mode = {
```

Comment:
should we also assign names for them??
I used:

motorDirectionReversed: BitFlag(0),
    calibrationMode: BitFlag(1),
    maintenanceMode: BitFlag(2),
    ledFeedback: BitFlag(3),

Reply:
Yes, I did for the others in WC but missed this one